arviz/stats/stats_utils.py

```diff
@@ -439,6 +439,10 @@ def get_log_likelihood(idata, var_name=None):
    (0.7, 1]   (bad)      {{4:{0}d}} {{8:6.1f}}%
    (1, Inf)   (very bad) {{5:{0}d}} {{9:6.1f}}%
 """
+SCALE_WARNING_FORMAT = """
+The scale is now log by default. Use 'scale' argument if you rely on a specific value. 

```

Comment:
could we use `Use 'scale' argument or 'stats.ic_scale' rcParam if...`? Same goes for warning in compare